tests/_test_msui/test_remotesensing.py

```diff
@@ -27,17 +28,250 @@
 
 from mslib.msui.remotesensing_dockwidget import RemoteSensingControlWidget
 import skyfield_data
+from PyQt5 import QtWidgets
+from mslib.msui import mpl_qtwidget as qt
+import datetime
+import sys
+from unittest.mock import Mock
+import matplotlib as mpl
+from matplotlib.collections import LineCollection
+app = QtWidgets.QApplication(sys.argv)
```

Comment:
see test results, this won't work
https://github.com/Open-MSS/MSS/actions/runs/4713789246/jobs/8359708435?pr=1739#step:9:30